src/styles/main.scss

```diff
@@ -1,3 +1,18 @@
-body {
+* {
+  font-family: "Roboto", sans-serif;
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your web document